acceptance/tests/resource/file/ticket_6448_file_with_utf8_source.rb

```diff
@@ -0,0 +1,82 @@
+require 'puppet/acceptance/environment_utils'
+extend Puppet::Acceptance::EnvironmentUtils
+
+test_name 'Ensure a file resource can have a UTF-8 source attribute, content, and path when served via a module' do
+  skip_test 'requires a master for serving module content' if master.nil?
+
+  tmp_environment = mk_tmp_environment_with_teardown(master, File.basename(__FILE__, '.*'))
+  agent_tmp_dirs = {}
+
+  agents.each do |agent|
+    agent_tmp_dirs[agent.hostname] = agent.tmpdir(tmp_environment)
+  end
+
+  teardown do
+    step 'remove all test files on agents' do
+      agents.each { |agent| on(agent, "rm #{agent_tmp_dirs[agent.hostname]}", :accept_all_exit_codes => true) }
```

Comment:
Yeah, it does. I didn't catch it in the run output linked above probably because it didn't fail the test.
```
* remove all test files on agents

  vki1nsgskq6wndd.delivery.puppetlabs.net (ubuntu1604-64-1) 12:01:44$ rm /tmp/ticket_6448_file_with_utf8_source_5gd3rak8.jL7rlB
    rm:     cannot remove '/tmp/ticket_6448_file_with_utf8_source_5gd3rak8.jL7rlB'    : Is a directory
```